apps/dataflow/src/scraper/fms/s3_utils.py

```
@@ -0,0 +1,248 @@
+from minio import Minio
```

Ruff (0.14.14)

[warning] Missing docstring in public module

(D100)